### PR TITLE
nixos: Add python3-pip

### PR DESCRIPTION
**Pull request by kjeremy**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -7035,6 +7035,7 @@ python3-pil:
 python3-pip:
   debian: [python3-pip]
   fedora: [python3-pip]
+  nixos: [python3Packages.pip]
```
Comment (ivanpauno) [inline rosdep/python.yaml:7038]: does that work?
I couldn't find that package on nixos.

I did find https://search.nixos.org/packages?channel=20.09&from=0&size=50&sort=relevance&type=packages&query=python38Packages.pip for nixos 20.09 and https://search.nixos.org/packages?channel=21.05&show=python39Packages.pip&from=0&size=50&sort=relevance&type=packages&query=python39Packages.pip for nixos 21.05.

Comment (kjeremy) [reply rosdep/python.yaml:7038]: Yes it does. See https://nixos.org/manual/nixpkgs/stable/#python section `15.21.2.3. Building packages and applications`

Comment (ivanpauno) [reply rosdep/python.yaml:7038]: Thanks for the info, lgtm then

**Review by ivanpauno — APPROVED** [review]

**Comment by ivanpauno** [comment]
See https://github.com/ros/rosdistro/pull/30589#issuecomment-909664339

**Review by hidmic — APPROVED** [review]

**Pull request merged**